### PR TITLE
tpm2_duplicate: update options

### PR DESCRIPTION
**Pull request by williamcroberts**
- -g becomes -G
- \--duplicate-key-private becomes \--privfile

Signed-off-by: William Roberts <william.c.roberts@intel.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Changelog
 ### next
+  * tpm2_duplicate: short and long options changes to make it consistent with
+    similair tools.
+      - -g becomes -G
+      - \--duplicate-key-private becomes \--privfile
```
Comment (idesai) [inline CHANGELOG.md]: @williamcroberts we have called this simply "private" elsewhere.

File: man/tpm2_duplicate.1.md [hunk 2]
```diff
@@ -34,11 +34,11 @@ These options control the key importation process:
 
     Specifies the context object for the parent key. Either a file, a handle number or null to select TPM2_RH_NULL.
 
-  * **-r**, **\--duplicate-key-private**=_FILE_:
+  * **-r**, **\--privfile**=_FILE_:
```
Comment (idesai) [inline man/tpm2_duplicate.1.md]: @williamcroberts we have called this simply "private" elsewhere.

File: tools/tpm2_duplicate.c [hunk 3]
```diff
@@ -115,16 +115,16 @@ bool tpm2_tool_onstart(tpm2_options **opts) {
 
     const struct option topts[] = {
       { "auth-key",              required_argument, NULL, 'p'},
-      { "inner-wrapper-alg",     required_argument, NULL, 'g'},
-      { "duplicate-key-private", required_argument, NULL, 'r'},
+      { "inner-wrapper-alg",     required_argument, NULL, 'G'},
+      { "privfile",              required_argument, NULL, 'r'},
```
Comment (idesai) [inline tools/tpm2_duplicate.c]: @williamcroberts we have called this simply "private" elsewhere.

File: man/tpm2_evictcontrol.1.md [hunk 4]
```diff
@@ -74,18 +74,18 @@ within the TPM. If an object is *persisted* then the object is resident at the
 
 ## To make a transient handle persistent
 ```
-tpm2_evictcontrol -a o -c object.context -p 0x81010002 -P abc123
+tpm2_evictcontrol -C o -c object.context -p 0x81010002 -P abc123
```
Comment (idesai) [inline man/tpm2_evictcontrol.1.md]: @williamcroberts can 
"-c object.context" become an argument. --> This is the input object worked on
 "-p <persistent-handle>" become "-c <persistent-handle>" --> This child object requires parent (-C) with auth (-P)

File: tools/tpm2_duplicate.c (hunk 3 above)
Comment (williamcroberts) [reply tools/tpm2_duplicate.c]: possibly. That's fine by me. We just need to make sure we mark this in our audit so we are consistent everywhere.

**Pull request merged**